lib/rubocop/cop/style/redundant_return.rb

```diff
@@ -23,7 +23,8 @@ module Style
       class RedundantReturn < Cop
         include OnMethodDef
 
-        MSG = 'Redundant `return` detected.'.freeze
+        MSG = 'Redundant `return` detected.' \
```

Comment:
Shouldn't this be the message only when you're actually returning multiple values?